pylinac/core/profile.py

```diff
@@ -210,6 +236,57 @@ def fwxm_center(self, x: int=50, interpolate: bool=False) -> Tuple[NumberLike, N
             fwxm_center_idx = int(round(fwxm_center_idx))
             return fwxm_center_idx, self.values[fwxm_center_idx]
 
+    @argue.bounds(x=(0, 100))
+    @argue.options(norm=('max', 'max grounded', 'cax', 'cax grounded'), interpolate=(True, False))
+    def distance_to_dose(self, x: int=50, norm='max grounded', interpolate=True) -> Tuple[float, float]:
```

Comment:
I don't understand this method name clearly. Distance to dose is usually associated with gamma. Isn't this just a FWXM call with normalization applied? Seems like if we add a normalization parameter to fwxm this would fix the problem. If you included the distance of left/right from the profile center this name might fit better. This also seems like it would simplify the functions later on in the fieldparams.py file.

Reply:
Yes, it is just a FWXM, but I didn't want to change the existing FWXM as I didn't know what else used it so I needed another name and distance_to_dose is the best I could come up with. It will be great if we can add a normalisation parameter to FHXM, but as you can see from the amount of code in distance_to_dose it is quite a minefield.

Reply:
How do you feel about calling it FWXM_indices as it returns the left and right index of the X dose level?

Reply:
This could probably be more easily accomplished with added parameters to field_edges to let the user set the height parameter instead of assuming 50% and adding in your normalization parameter? Setting good defaults would then not change outputs for upgrading users. 
https://github.com/jrkerns/pylinac/blob/master/pylinac/core/profile.py#L262

Reply:
I've implemented this, and it does simplify things considerably, but I've got bogged down in small round off errors that lead to 1 more or 1 less field edge values being selected which has quite a large effect on the symmetry. In your original field_edges function (now called field_edges_deprecated) you call fwxm_center with interpolation effectively set to false causing fwhmc to be rounded off. Calling this function with interpolation set to True gives the same results as my new field_edge function. To my mind this gives a more accurate result, but you may have had other reasons for calling fwxm_center with interpolation set to False.
I'll commit the changes and upload so you can take a look.